docs/portal/image-registry/viewing-projects.md

```diff
@@ -9,9 +9,9 @@ The following procedure describes how to view information about projects. Button
 
 | **Element Number**       | **Description**                               |
 | -------------------------|-----------------------------------------------| 
-1                                                                                                       | The Basic Information area shows basic information about the project.                                             |
-| 2                        | Click this button to return to the Registry Projects page.                                                             |
-| 3                        | Click this button to view the [logs](</docs/portal/image-registry/viewing-logs.md>) associated with the project.                                                          |
-| 4                        | Click this button to [delete](</docs/portal/image-registry/deleting-projects.md>) the project.      |
-| 5                        | Click this button to [update](</docs/portal/image-registry/updating-projects.md>) the project.      |
-| 6                        | The Repositories area shows repositories associated with the project. Click a number in the **Tags** column to display [detailed tag information](</docs/portal/image-registry/managing-tags-under-a-repository.md>).                                               |
\ No newline at end of file
+<p style="text-align: center;">1</p>                                                                                                       | The Basic Information area shows basic information about the project.                                             |
```

Comment:
The preview shows there are errors. Please check.
![image](https://user-images.githubusercontent.com/43203744/105330152-ee299e80-5c0c-11eb-80d7-6bb4d449a695.png)